scripts/support/create-support-bundle.bat

```diff
@@ -0,0 +1,70 @@
+:: Copyright  observIQ, Inc.
+::
+:: Licensed under the Apache License, Version 2.0 (the "License");
+:: you may not use this file except in compliance with the License.
+:: You may obtain a copy of the License at
+::
+::      http://www.apache.org/licenses/LICENSE-2.0
+::
+:: Unless required by applicable law or agreed to in writing, software
+:: distributed under the License is distributed on an "AS IS" BASIS,
+:: WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+:: See the License for the specific language governing permissions and
+:: limitations under the License.
+@echo off
+setlocal enabledelayedexpansion
+
+:: Default directory for logs
+set "collector_dir=C:\Program Files\observIQ OpenTelemetry Collector"
```

Comment:
This is probably fine in 99% of cases, but I think we could be more robust here by calling the registry key for the install directory here:
```bat
FOR /F "tokens=2*" %%A IN ('reg query "HKLM\Software\Microsoft\Windows\CurrentVersion\Uninstall\observIQ Distro for OpenTelemetry Collector" /v "InstallLocation"') DO set "collector_dir=%%~B"
```

I do know customers have asked about installing in different locations before, so I think it'd be useful.